changelog.md

```diff
@@ -29,6 +42,24 @@ Since this topic has so distinct requirements for different organizations, it is
 This version now implements all the necessary hooks needed to secure most external interfaces to your specific needs.
 If you are interested in a very simple implementation, you can look into the [Example Auth Plugin](https://github.com/mesosphere/marathon-example-plugins/tree/master/auth)
 
+#### Persistent Store Cache 
+All entities in the persistent store (ZK) are loaded into a cache during leader election.
+Subsequent reads are delivered from that cache. Updates to entities also update the cache.
+This cache should improve read access time significantly.
+Please be aware that the time until the leader takes over leadership will be increased with a lot of apps and tasks. 
```

Comment:
After the current leader loses his leadership, there's a leaderless lapse of time until the next leader is elected. During that time Marathon is leaderless and no API calls are accepted.

The new behaviour makes an elected candidate load the cache before becoming leader, which means that the duration of the leaderless period can be longer if there are many entities to load.